dashboard/cron.py

```diff
@@ -246,18 +249,9 @@ def update_with_bq_access(self):
         # BQ Total Bytes Billed to report to status
         total_bytes_billed = 0
 
-        # the earliest start date of all courses
-        course_start_time = utils.find_earliest_start_datetime_of_courses()
-
-        # the earliest latest date of all resources_accessed
-        try:
-            latest_resource = ResourceAccess.objects.latest('access_time')
-            latest_resource_time = latest_resource.access_time
-        except ResourceAccess.DoesNotExist:
-            latest_resource_time = course_start_time
-
-        logger.info(f"Earliest Start: {course_start_time} Latest resource: {latest_resource_time}")
+        data_last_updated = Course.objects.get_data_last_updated()
```

Comment:
Does this need to be limited to the locked course list also?

Reply:
Yeah, good catch.